content/audio.md

```diff
@@ -17,50 +21,98 @@ section: software-troubleshooting
 tableOfContents: true
 ---
 
-Sound settings or packages related to the sound system can become corrupt or broken. Many times, deleting the configuration files, reinstalling the sound-related packages, and restarting the audio software can help. These commands can also help fix the <u>Sound</u> settings showing "Dummy Output" as the audio output.
+If your system has no sound, distorted sound, or missing input/output devices, this guide will walk you through verifying hardware, reloading drivers, and resetting audio services for Pop!_OS 22.04 and Pop!_OS 24.04.
 
-## Reset Audio Software (server)
+## Verify if the Correct Device is Selected
 
-If the system is not playing audio, first try restarting the audio daemon:
+If the system is not playing any audio, you can confirm the selected device from the System Settings.
 
-| PulseAudio | PipeWire |
-| :--------- | :-------|
-| Ubuntu pre-22.10 and Pop!\_OS pre-22.04 | Ubuntu 22.10+ and Pop!\_OS 22.04+ |
-| `systemctl --user restart pulseaudio` | `systemctl --user restart wireplumber pipewire pipewire-pulse` |
-| `rm -r ~/.config/pulse` |  `rm -r ~/.config/pulse` |
-| `pulseaudio -k` | |
+Go to <u>Launcher</u> and search for Sound:
 
-This set of commands first restarts the sound daemon and removes the user's configuration for PulseAudio. On systems still using PulseAudio as a server, it restarts the PulseAudio server, which will create new default audio configuration files.
+![audio_1](images/audio/audio_1.png)
+
+Under the Sound Settings, the Input and Output Device selected will be displayed:
+
+![audio_2](images/audio/audio_2.png)
+
+## Verify Audio Devices Detected by the System
+
+Below are set of commands that can be used to detect various information about your audio device.
+
+List all detected audio devices:
+
+```
+aplay -l
+```
+
+List audio input devices:
+
+```
+arecord -l
+```
+
+List the current sound cards:
+
+```
+cat /proc/asound/cards
+```
+
+List kernel modules related to audio:
+
+```
+sudo lspci -k | grep -A 3 -i audio
+```
+
+Your audio devices should appear in these lists.
+
+If not, this may be a driver issue.
```

Comment:
Please combine this sentence with the line above it so it's a single paragraph.

I'd slightly rephrase the last sentence to `If not, there may be a driver issue.`